examples/react/UIShell/src/App.tsx

```diff
@@ -194,8 +194,8 @@ function App() {
             </Theme>
             <Theme theme="g100">
               <SideNav
-                navType={SIDE_NAV_TYPE.PANEL}
                 isChildOfHeader={false}
+                isTreeview={true}
```

Comment:
```suggestion
```
I don't think we want this for this secondary nav?

Reply:
woops, yeah i'll remove